StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/SavedPaymentMethods/SavedPaymentMethodsSheet.swift

```diff
@@ -0,0 +1,233 @@
+//
+//  SavedPaymentMethodSheet.swift
+//  StripePaymentSheet
+//
+//
+//  ⚠️🏗 This is feature has not been released yet, and is under construction
+//  Note: Do not import Stripe using `@_spi(STP)` or @_spi(PrivateBetaSavedPaymentMethodsSheet) in production.
+//  Doing so exposes internal functionality which may cause unexpected behavior if used directly.
+//
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+@_spi(STP) @_spi(PrivateBetaSavedPaymentMethodsSheet) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+@frozen public enum SavedPaymentMethodsSheetResult {
+    /// The customer completed the payment or setup
+    /// - Note: The payment may still be processing at this point; don't assume money has successfully moved.
```

Comment:
Does this comment still make sense for SPMS? We're not completing a payment.

Reply:
Comment doesn't make sense, and actually, we don't need to make this public -- it is purely an object we use for confirming an intent from savedpaymentmethodssheet.